git-pathspec/tests/pathspec.rs

```diff
@@ -0,0 +1,11 @@
+#[test]
```

Comment:
It's good to put down some tests you think will be implemented some time!

Ideally these are ignored while they serve as markers to not forget them, without breaking CI (_we never want to break it and fix it swiftly if it happens anyway_).